### ISSUE TITLE
NameError: global name 'die' is not defined

### ISSUE THREAD

**Issue by jordanbtucker** [tabatkins/bikeshed#1093]
I attempted to use the API and got the following error. I figured it was just an issue with the API, so I installed bikeshed locally and got the same error. This error occurs even with the --force flag set.

```
Traceback (most recent call last):
  File "/sites/api.csswg.org/bikeshed/bikeshed.py", line 5, in 
    bikeshed.main()
  File "/sites/api.csswg.org/bikeshed/bikeshed/cli.py", line 200, in main
    doc.preprocess()
  File "/sites/api.csswg.org/bikeshed/bikeshed/Spec.py", line 115, in preprocess
    self.md.computeImplicitMetadata(doc=self)
  File "/sites/api.csswg.org/bikeshed/bikeshed/metadata.py", line 143, in computeImplicitMetadata
    self.status = config.canonicalizeStatus(self.rawStatus, self.group)
  File "/sites/api.csswg.org/bikeshed/bikeshed/config/status.py", line 173, in canonicalizeStatus
    die("{0}", msg)
NameError: global name 'die' is not defined
```

**Comment by jordanbtucker** [comment]
Looks like it was because I was using the `ED` status without a group. Using `w3c/ED` works. Still a more useful error message would be helpful.

**Comment by tabatkins** [comment]
Yeah, that's a programming error on my part, not an error message meant for you. ^_^ Sorry, did some heavy refactoring of the codebase into more highly separated modules a little bit ago, and Python's lack of a typesystem means it's easy for these sorts of things (a required method is no longer imported) to slip by if you're not exercising that code path...

Fixed in 700cc55f, sorry for the trouble.

(And if you're using a W3C group that isn't being recognized, please let me know so I can add it to the list of Groups automatically allowed to use the W3C statuses.)
